packages/file-search/src/common/file-search-service.ts

```diff
@@ -54,3 +54,5 @@ export namespace FileSearchService {
         defaultIgnorePatterns?: string[]
     }
 }
+
+export const WHITESPACE_QUERY_SEPARATOR = ' ';
```

Comment:
We may want something more sophisticated than this. Other language settings have different default whitespace characters, so I'd suggest regex whitespace `/\s+/`, unless there's a good reason not to use it?